antiphishing/antiphishing.py

```
@@ -1,357 +1,765 @@
+        return message.guild.me.guild_permissions.moderate_members
+
+    async def _notify_action(self, message: discord.Message, domain: str, is_fallback: bool = False):
+        """Sends a warning message in the channel."""
+        if not await self._can_notify(message):
+            log.warning(f"Missing SEND_MESSAGES permission for notify action in {message.channel.name} ({message.guild.name}).")
+            return
+
+        try:
+            staff_role_id = await self.config.guild(message.guild).staff_role()
+            staff_mention = f"<@&{staff_role_id}>" if staff_role_id else ""
+            allowed_mentions = discord.AllowedMentions(roles=True if staff_role_id else False)
```

Ruff (0.8.2)

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)